Лабораторная работа №1/Расчёт количества книг на дискете.py

```diff
@@ -0,0 +1,12 @@
+# TODO Найдите количество книг, которое можно разместить на дискете
+i = 0
+kol = 0 #кол-во книг
+V = 50 * 25 * 4 * 100 #объём одной книги
```

Comment:
Отличная работа!
Классное применение цикла!
Но будет немного лучше, если Вы избавитесь от "магических" чисел, создав несколько переменных под каждое число в задаче.
Например:
```suggestion
disk_mb = 1.44
pages = 100
lines_per_page = 50
```
И так далее.
Таким образом, ваш код будет более понятен человеку, который не знает суть Вашего задания. Код станет более читаемым.